test/test_lb.py

```diff
@@ -122,7 +122,7 @@ def test_nat_to_lb_nat(request, prepare_ipv4, grpc_client, port_redundancy):
 
 	grpc_client.delnat(VM1.name)
 	grpc_client.dellbtarget(lb_name, lb_vm1_ul_ipv6)
-	grpc_client.dellbprefix(VM1.name, lb_ip)
+	grpc_client.dellbprefix(VM1.name, f"{lb_ip}/32")
```

Comment:
If these functions are now only ever used like this, can you define a `lb_pfx`  constant instead? Or alternatively rename them to `addlbip` and `dellpip` and do the `f{lb_ip}/32` inside?

Reply:
Done